### DISCUSSION TITLE
[request] give option so first up scroll holds chat auto scroll

### DISCUSSION THREAD

**Discussion by lastmetroid** [night/betterttv#6934] [Ideas]
Please give optional option so that the first up scroll in chat will hold the auotscroll. it is annoying that it just starts to happen to pause after the 2th or 3rd scroll and causes issues if the chat scrolls too quickly.

**Comment by night** [comment]
Because BetterTTV is a tweak on Twitch's native chat, we don't hold control over Twitch's auto scrolling. We have some ability to manipulate their resulting scroll handler, but we cannot change logic which controls how it pauses and unpauses.
